packages/boxel/addon/components/boxel/progress-icon/index.gts

```diff
@@ -0,0 +1,65 @@
+import Component from '@glimmer/component';
+import { EmptyObject } from '@ember/component/helper';
+import { htmlSafe } from '@ember/template';
+import { SafeString } from '@ember/template/-private/handlebars';
+import { svgJar } from '@cardstack/boxel/utils/svg-jar';
+
+import cn from '@cardstack/boxel/helpers/cn';
+import not from 'ember-truth-helpers/helpers/not';
+import or from 'ember-truth-helpers/helpers/or';
+import '@cardstack/boxel/styles/global.css';
+import './index.css';
+
+interface Signature {
+  Element: HTMLDivElement;
+  Args: {
+    isCancelled?: boolean;
```

Comment:
Unrelated to the PR, the spelling of this is something we tried to [standardise on](https://github.com/cardstack/cardstack/commit/f90762ea7feab2ca50893541106a53f172a44396), but thankfully Glint should make it easier to fix now!